.circleci/config.yml

```diff
@@ -231,6 +231,9 @@ jobs:
           tag: latest
           use_tarfile: true
       - release-management/create-github-release
+    environment:
+      # Fingerprint for developer.salesforce.com
+      SFDX_DEVELOPER_TRUSTED_FINGERPRINT: "3E:6C:CA:83:EA:88:92:E0:6C:7C:5C:ED:43:A6:0A:6C:33:74:5E:7F"
```

Comment:
Actually, I'm a little confused. From the work item description, this value seemed like it would be stored in CircleCi project variable. Is it okay to store this fingerprint in plain text?

Reply:
Closing the loop based on slack discussion, this is the public cert fingerprint of developer.salesforce.com. It is used for certicate pinning.